opencraft/settings.py

```diff
@@ -482,6 +482,13 @@
 # Enable or disable celery heartbeats on instances managed by Ocim
 EDX_WORKERS_ENABLE_CELERY_HEARTBEATS = env.bool('EDX_WORKERS_ENABLE_CELERY_HEARTBEATS', default=False)
 
+# Open EdX Instance custom theme for clients ##################################
+EDXAPP_COMPREHENSIVE_THEME_SOURCE_REPO = env(
```

Comment:
@giovannicimolin, just asking to be sure. These can be overridden in the extra settings right? Also, can the settings variables be renamed to indicate that they are related to the simple-theme? Right now it is not clear what they are used for and where.

Reply:
@lgp171188 Yes, we can always override those. 
I don't think it's good for us to change the names because those are the same that Ansible uses. I can change them to `CUSTOM_THEME_REPO` and `CUSTOM_THEME_VERSION` if you'd like, but those are new variable names. Do you have any better suggestion on this?

Reply:
@lgp171188 I changed the variable names to `SIMPLE_THEME_SKELETON_THEME_REPO` and `SIMPLE_THEME_SKELETON_THEME_VERSION` and added a small description to the variables.